<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BmpToolDialog</class>
 <widget class="QDialog" name="BmpToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BMP Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Rules</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="Tiled::Internal::MixedTilesetView" name="tableView">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Brush size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="brushSize">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>300</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="brushSquare">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="tiled.qrc">
             <normaloff>:/images/22x22/brush-square.png</normaloff>:/images/22x22/brush-square.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="brushCircle">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="tiled.qrc">
             <normaloff>:/images/22x22/brush-circle.png</normaloff>:/images/22x22/brush-circle.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QToolButton" name="toggleOverlayLayers">
         <property name="text">
          <string>Show/Hide FloorOverlay Layers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showMapTiles">
         <property name="text">
          <string>Show map tiles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="rulesBox">
         <property name="title">
          <string>Rules.txt</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="rulesFile">
            <property name="text">
             <string>&lt;none&gt;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QToolButton" name="reloadRules">
              <property name="text">
               <string>Reload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="importRules">
              <property name="text">
               <string>Import...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="exportRules">
              <property name="text">
               <string>Export...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="blendsBox">
         <property name="title">
          <string>Blends.txt</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="blendsFile">
            <property name="text">
             <string>&lt;none&gt;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QToolButton" name="reloadBlends">
              <property name="text">
               <string>Reload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="importBlends">
              <property name="text">
               <string>Import...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="exportBlends">
              <property name="text">
               <string>Export...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tiled::Internal::MixedTilesetView</class>
   <extends>QTableView</extends>
   <header>BuildingEditor/mixedtilesetview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tiled.qrc"/>
 </resources>
 <connections/>
</ui>
